ecommerce_integrations/woocommerce/woocommerce_connection.py

```diff
@@ -172,27 +170,32 @@ def rename_address(address, customer):
 def link_items(items_list, woocommerce_settings, sys_lang):
 	for item_data in items_list:
 		item_woo_com_id = cstr(item_data.get("product_id"))
-
-		if not frappe.db.get_value("Ecommerce Item", {"erpnext_item_code": item_woo_com_id}, "name"):
+		tmp_erpnext_item_code = _("woocommerce - {0}", sys_lang).format(item_woo_com_id)
+		if not frappe.db.get_value(
+			"Ecommerce Item", {"erpnext_item_code": tmp_erpnext_item_code}, "name"
+		):
+			if not frappe.db.get_value("Item", {"woocommerce_id": item_woo_com_id}, "name"):
+				create_erp_item(
+					item_woo_com_id, woocommerce_settings, item_data, sys_lang, tmp_erpnext_item_code
+				)
 			item = frappe.new_doc("Ecommerce Item")
 			item.integration = MODULE_NAME
-			if not frappe.db.get_value("Item", {"woocommerce_id": item_woo_com_id}, "name"):
-				create_erp_item(item_woo_com_id, woocommerce_settings, item_data, sys_lang)
-			item.erpnext_item_code = item_woo_com_id
+			item.erpnext_item_code = tmp_erpnext_item_code
 			item.integration_item_code = item_data.get("sku")
-			item.has_variants: 0  # to check
+			item.has_variants = 0  # to check
 			item.sku = item_data.get("sku")
 			item.flags.ignore_mandatory = True
 			item.save(ignore_permissions=True)
 
 
-def create_erp_item(item_woo_com_id, woocommerce_settings, item_data, sys_lang):
+def create_erp_item(item_woo_com_id, woocommerce_settings, item_data, sys_lang, erpnext_item_code):
```

Comment:
Can this function be somehow reused?

https://github.com/frappe/ecommerce_integrations/blob/6e4af84c2b9b02603b95890e752086ab2e1d9ad6/ecommerce_integrations/ecommerce_integrations/doctype/ecommerce_item/ecommerce_item.py#L127-L135

Reply:
Will refactor to use Ecommerce_item create method.